ark/segmentation/marker_quantification.py

```diff
@@ -41,6 +41,12 @@ def get_single_compartment_props(segmentation_labels, regionprops_base,
         props_options=list(REGIONPROPS_FUNCTION.keys())
     )
 
+    # if image is just background, return empty df
+    if len(np.unique(segmentation_labels)) < 2:
+        output_list = regionprops_base + regionprops_single_comp
+        blank_df = pd.DataFrame(columns=output_list)
+        return blank_df
+
```

Comment:
Should we add a warning to notify the user that there are no cells within the image they've supplied in addition to returning an empty dataframe?